Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,10 +1,170 @@
 # Multiplayer Samples Co-op Changelog
 
+## [0.2.0] - 2021-05-19
+
+v0.2.0 is an Early Access release for Multiplayer Samples Co-op.
+
+It requires and supports Unity v2020.3.8f1 LTS and Unity MLAPI v0.1.0. For additional information on MLAPI, see the changelog and release notes for the Unity MLAPI package.
+
+### New features
+
+* Introduced static scene `NetworkObject`s to Boss Room including the following updates:
+
+    * Implemented a `ScriptableObject` based event system to encapsulate events inside assets. These objects include a `GameEvent` (ScriptableObject) and `GameEventListener` (MonoBehaviour) to encapsulate events inside assets, located in the `ServerBossRoomState` prefab which now has a `GameEventListener` component. The event associated to this listener is `BossDefeated`, which the Boss raises when the `LifeState` is Dead in the `RaiseEventOnLifeChange` component.
+    * Added two separator `GameObject`s for scene readability: runtime `NetworkObject`s and `NetworkObject`s already placed in the scene.
+    * Added a custom editor for GameEvents to fire in the editor (greatly enhances testing).
+    * The `LifeState` NetworkVariable was moved from `NetworkCharacterState` into its own component, `NetworkLifeState`.
+    * Cleaned up and removed old spawn prefab collections and spawner scripts (`NetSpawnPoint`).
+
+* Updated the user interface including the following:
+
+  * When joining a game, a "Connecting..." UI loads. When disconnecting from a game, you are returned to the MainMenuScene with a "Connection to Host lost" message. If the game fails to connect, a general message "Connection to Host failed" loads. 
+  * Added an option to leave the lobby after joining it. You no longer have to restart the game to leave the lobby.
+  * Added option to cancel the game after clicking **Ready**. 
+  * Added a gear icon for accessing and modifying app settings. 
+  * The UI now tracks and displays player has arrived, number of connected players in the lobby, and player status. 
+
+* Added F/X and animation assets for the game including:
+
+  * Audio files for boss sound effects
+  * Ramp-up animation for hero movement and actions
+  * Visual effects and particles for Tank charge skill 
+  * Art assets to wave spawner, including animations for ReceiveDamage, Broken (died), and Revive 
+  * Game music 
+
+* Added audiomixer to Boss Room for separate audio channels and a master mixer. 
+
+* Updated and added various hero abilities:
+
+  * Added a cooldown to Archer's PowerShot. 
+  * Added the Rogue's Dagger and Dash skills. 
```

Comment:
```suggestion
  * Added the Rogue's Dagger and Dash skills. The dash skill shows an install teleport (using an RPC) instead of a charge like the boss' (which updates its position over time). 
```